Moderation-Policy.md

```diff
@@ -75,6 +76,26 @@ subject to Moderation.
 The Moderation Team is responsible for deciding what constitutes inappropriate
 behavior that may be subject to Moderation.
 
+## Outside Commentary via Social Media
+
+Comments by individuals made in any venue falling outside the explicit scope of
+responsibility of the Node.js Foundation, TSC, or Community Committee are
+explicitly not covered by this policy unless the individual is explicitly acting
+or commenting on behalf of the Node.js Foundation, TSC or Community Committee
+in an official capacity. The statements, views, and opinions contained in such
+comments are not endorsed by, nor do they necessarily reflect, the opinions and
+views of the Node.js Foundation, TSC or Community Committee.
+
+Moderation requests based solely on the content of comments made in any such
+venue *may* be ignored.
+
+All Collaborator's are cautioned, however, that any statements made publicly by
+members of the Node.js GitHub Organization *will* be interpreted by readers
+as reflecting on the project as a whole, despite the statement above. It is
+therefore important for Collaborators to remain cognizant of how their
+statements will be interpreted and what impact such statements may have on
+the project.
```

Comment:
Thank you for this line. Humans are not robots and it is unreasonable to expect them to separate the person from their specific role. We've seen how this plays out in other situations.

I wonder if this sentence can be strengthened to include something about social posts potentially leading to ramifications for their status on the committee even if it's not a strict CoC violation.

Meaning, e.g. you can still be asked to resign, or whatever it is Node does as a punitive action these days,  even if it's not under the banner of CoC enforcement.